### ISSUE TITLE
checkpoint error

### ISSUE THREAD

**Issue by matheuscass** [tuckerdarby/tf-deeplab-v3#3]
When I run inference.py I'm having the following trouble:
2018-07-17 17:50:08.094544: W C:\tf_jenkins\workspace\rel-win\M\windows\PY\36\tensorflow\core\framework\op_kernel.cc:1202] OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key 3a/BatchNorm/beta not found in checkpoint

Any thoughts in how to proceed? =(

**Comment by tuckerdarby** [comment]
Hmmm. Did you try running training first?

**Comment by matheuscass** [comment]
Yep! I started the training and I forced an interruption to see if the checkpoints were working. When I run train.py again, the training stops and nothing happens after 25 steps (number of steps to save a checkpoint). When I run inference.py with the most recent model saved, the error I described above happens...
